python_modules/dagster/dagster/core/selector/subset_selector.py

```diff
@@ -13,13 +13,17 @@
 MAX_NUM = sys.maxsize
 
 
+class LeafIgnoredNode:
+    """Marker for no further nesting ignore needed."""
```

Comment:
> no further nesting ignore needed

I find this wording confusing, consider more verbosity

Reply:
updated